app/layout_elements.py

```diff
@@ -20,36 +21,55 @@ def display_costs(
     key_suffix = key_suffix.lower().replace(" ", "_")
     st.subheader(titlestring)
 
-    # filter data:
-    df_res = df_costs.copy()
+    c1, c2 = st.columns(2)
 
-    # select filter:
-    show_which_data = st.radio(
-        "Select elements to display:",
-        ["All", "Manual select"],
-        index=0,
-        horizontal=True,
-        key=f"show_which_data_{key}_{key_suffix}",
-    )
+    # select which dataset to display:
+    if st.session_state["user_changes_df"] is not None:
+        with c2:

```

Comment:
I would prefer to put the user modification selection below and and not side by side

Reply:
I would prefer to leave it like it is for now